docs/README.md

```diff
@@ -0,0 +1,26 @@
+[![Build Status](https://api.travis-ci.org/daostack/arc.js.svg?branch=master)](https://travis-ci.org/daostack/arc.js)
+[![NPM Package](https://img.shields.io/npm/v/@daostack/arc.js.svg?style=flat-square)](https://www.npmjs.org/package/@daostack/arc.js)
+
+# DAOstack Arc.js
+
+[DAOstack Arc.js](https://github.com/daostack/arc.js) sits just above [DAOstack Arc](https://github.com/daostack/arc) on the DAO stack.  It is a library that facilitates javascript application access to the DAOstack Arc ethereum smart contracts.
+
+For more information about Arc contracts and the entire DAOstack ecosystem, please refer to the [Arc documentation](https://daostack.github.io/arc/README/).
+
+
+### Documentation
+Check out the [complete documentation on Arc.js](https://daostack.github.io/arc.js).  **Please note** that viewing documents directly in the Arc.js GitHub repository is not supported as a means of obtaining Arc.js documentation online, you will find that certain document functionality does not work when viewed that way.
+
+## Contribute to Arc.js
+
+PRs are welcome but please first consult with the [Contribution guide](https://github.com/daostack/arc/blob/master/CONTRIBUTING.md).
+
+Join us on [Slack](https://daostack.slack.com/)!
+
+Join us on [Telegram](https://t.me/daostackcommunity)!
+
+## Security
```

Comment:
This should be removed.

Reply:
@orenyodfat You mean the Telegram line?  Remove on the top-level readme too?

Reply:
it is enough to keep it on the top level readme only